### PR TITLE
Correctly pull issue number for pull request review events

### LINKED ISSUE #289: Issue Number does not populate correctly in `@actions/github` Context.issue
**Describe the bug**
Issue number is empty for PullRequestReview and PullRequestReviewComment events

**To Reproduce**
1. Create an action using the following ts file:
```
import * as core from '@actions/core'
import * as github from '@actions/github'

async function run(): Promise<void> {
  try {
    core.info(`Issue number is: ${JSON.stringify(github.context.issue.number)}`)
  } catch (error) {
    core.setFailed(error.message)
  }
}

run()
```
2.  Setup the workflow to run on `on: [pull_request_review, pull_request_review_comment]`
3. Create a PR with a linked issue

**Expected behavior**
The Issue number field should be populated.

**Screenshots**
The issue number field is null.

**Additional context**
We appear to be accessing `pullRequest` instead of `pull_request` causing this bug: 
https://github.com/actions/toolkit/blob/master/packages/github/src/context.ts#L47

### PR DESCRIPTION
**Pull request by thboop**
Resolves #289

### CODE REVIEW

**Comment by bryanmacfarlane** [comment]
Let's get this in since we're about to release an update github pkg for other work (proxy)

File: packages/github/src/context.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class Context {
 
     return {
       ...this.repo,
-      number: (payload.issue || payload.pullRequest || payload).number
+      number: (payload.issue || payload.pull_request || payload).number
```
Comment (ericsciple) [inline packages/github/src/context.ts]: @thboop a pull request is not an issue?

Comment (ericsciple) [reply packages/github/src/context.ts]: The issue linked to this PR indicates the \"linked issue\" is desired.

It looks like the change above returns the pull request number, not the inked issue number.

Comment (ericsciple) [reply packages/github/src/context.ts]: discussed offline. before only PR events works, this change adds PR comment and PR review comments

**Pull request merged**